pip/Cargo.toml

```diff
@@ -12,10 +12,15 @@ license.workspace = true
 [dependencies]
 num-bigint = { workspace = true }
 num-complex = { workspace = true }
-pyo3 = { workspace = true }
 qsc = { path = "../compiler/qsc" }
 miette = { workspace = true, features = ["fancy"] }
 
+[target.'cfg(not(any(target_os = "windows")))'.dependencies]
+pyo3 = { workspace = true, features = ["abi3-py37", "extension-module"] }
+
+[target.'cfg(any(target_os = "windows"))'.dependencies]
+pyo3 = { workspace = true, features = ["abi3-py37", "extension-module", "generate-import-lib"] }
+
```

Comment:
Could you add a comment so we later remember why we had to add the windows specific setting?